src/Smp.Web/ClientApp/src/app/profile/profile.component.ts

```diff
@@ -35,8 +36,8 @@ export class ProfileComponent implements OnInit {
   ) {  }
 
   ngOnInit() {
-    this.userId = this.route.snapshot.paramMap.get('id');
     let visitingUserId = this.globalHelper.localStorageItem<CurrentUser>('currentUser').id;
+    this.userId = this.route.snapshot.paramMap.get('id') ?? visitingUserId;
```

Comment:
I think this change might be irrelevant as we probably don't allow navigation to this page without auth data locally. This change is purely to match the null that get set in the nav footer.